pkg/webhook/v1beta1/experiment/validator/validator.go

```diff
@@ -311,8 +311,13 @@ func (g *DefaultValidator) validateTrialTemplate(instance *experimentsv1beta1.Ex
 
 		// Check if parameter reference exist in experiment parameters
 		if len(experimentParameterNames) > 0 {
-			if _, ok := experimentParameterNames[parameter.Reference]; !ok {
-				return fmt.Errorf("parameter reference %v does not exist in spec.parameters: %v", parameter.Reference, instance.Spec.Parameters)
+			// Check if parameter is trial metadata
+			regex := regexp.MustCompile(consts.TrialTemplateMetaReplaceFormatRegex)
+			match := regex.FindStringSubmatch(parameter.Reference)
+			if !(len(match) > 0 && contains(consts.TrialTemplateMetaKeys, match[1])) {
```

Comment:
I think `contains(contains(consts.TrialTemplateMetaKeys, match[1]))` returns `false` when `parameter.Reference` is `Anotations` or `Labels`. 
WDYT @alexeygorobets ?

https://github.com/kubeflow/katib/blob/e031b5eeb437e3966166c455666ba97c2b68a414/examples/v1beta1/trial-template/trial-metadata-substitution.yaml#L43-L48